### ISSUE TITLE
Default facade-x ns uri

### ISSUE THREAD

**Issue by enridaga** [SPARQL-Anything/sparql.anything#36]
Currently we are using urn:facade-x:ns# but it would be better to use a public one, e.g. http://sparql.xyz/facade-x/ns/

**Comment by enridaga** [comment]
Besides, we should distinguish between the namespace of the constants generated by the system (e.g. the type fx:Root) from the default namespace used for generated schema elements from the data (e.g. xyz:name).
What about the following two namespaces?
```
prefix fx: <http://sparql.xyz/facade-x/ns/>
prefix xyz: <http://sparql.xyz/facade-x/xyz/>
```

**Comment by luigi-asprino** [comment]
closed with commit 584aef0
